src/Microsoft.DotNet.XHarness.Common/Execution/ProcessManager.cs

```diff
@@ -157,13 +159,21 @@ static async Task KillTreeAsyncInternal(int pid, Action<int, int> kill, Func<ILo
                                 process: dbg,
                                 log: log,
                                 stdout: log,
-                                stderr:log,
+                                stderr: log,
                                 kill: kill,
                                 getChildrenPS: getChildrenPS,
                                 timeout: TimeSpan.FromSeconds(30),
                                 diagnostics: false);
                         }
                     }
+                    catch (Win32Exception e) when (e.Message == "No such file or directory")
```

Comment:
Checking the message isn't ideal since it could be localized. Better to check `NativeError == 2` which is the error code for file not found on Unix.